### PR TITLE
Some comments/edits

### PR DESCRIPTION
**Pull request by svahl991**
## Description

This PR is just intended to be a communication of some comments and edits I found during a review of the 4DVar tutorial. I don't know if it's useful for making actual changes because other parts of the file got modified as well.

### CODE REVIEW

File: notebooks/qg4Dvar_tutorial.ipynb [hunk 1]
```diff
@@ -34,7 +34,7 @@
...
+    "in which $H(..)$ is the so-called observation operator that maps a state $x$ {\\em(?) at the beginning of the time window} to observation space. Observation space in this case contains observations valid over the whole time window. This means that to compare the model state with the observations $H$ has to first bring the model state to the observation time, and then extract from the state at that time the corresponding modelled observation. {\\em(?) This means that $H$ now contains the model operator.} The $\\epsilon_{true}$ is the observation error. This error is due to the measurement process itself: every measurement will have an error related to the precision of the measurement. There is also another part to this error, called the representation error, that we will not discuss here as it is not essential for understanding 4DVar. <br><br>\n",
```
Comment (svahl991) [inline notebooks/qg4Dvar_tutorial.ipynb:37]: This `\em` markup doesn't seem to be getting rendered correctly. Not sure what is intended or correct. I just marked with a `(?)` here and elsewhere.

File: notebooks/qg4Dvar_tutorial.ipynb [hunk 2]
```diff
@@ -48,14 +48,14 @@
...
     "<br><br>\n",
     "<center>$J(\\delta x) = (\\delta-\\xi^{(i)}) x^TB^{-1}(\\delta-\\xi^{(i)}) + \\left(H\\delta x - d\\right)^T R^{-1}\\left(H\\delta x - d\\right)$ </center> <br>\n",
     "in which $\\delta x = x - x^{(i)}$, where $x^{(i)}$ is the vector $\\xi^{(i)}=x_b-x^{(i)}$ and $H$ is now the linearized observation operator, linearized along the nonlinear trajectory of the model from starting point $x_b^{(i)}$. The observation is transformed to $d=y-H(x^{(i)})$.<br>\n",
     "\n",
-    "Each of these incremental cost functions have the same form as when $H$, so the model and the mapping to observation space at observation time, is linear. Hence, we can write for the solution:\n",
+    "Each of these incremental cost functions have the same form as when $H$, so the model and the mapping to observation space at observation time, is linear(?). Hence, we can write for the solution:\n",
```
Comment (svahl991) [inline notebooks/qg4Dvar_tutorial.ipynb:58]: I found this sentence confusing. I think there's a grammar error somewhere, but it's possible I just don't understand it.

File: notebooks/qg4Dvar_tutorial.ipynb [hunk 3]
```diff
@@ -79,13 +79,37 @@
...
+    "The way we solve this set of equations is as follows. First, we set all $\\lambda_i=0$ and use the first two equations to run the first guess from the start to the end of the time window. Then we use the last two equations to integrate the adjoint variable $\\lambda_i$ from the end of the time window back to the beginning of the time window. These backward-in-time equations are also called the {\\bf(?) adjoint equations}. This is because they contain $M_t^T$, which is the adjoint of the tangent-linear forward model $M_t$ at each time $t$. These adjoint equations have a forcing term at each time $t$ that is equal to the innovation at that time $t$. This is the mechanism with which observation information is brought back to the beginning of the window. \n",
```
Comment (svahl991) [inline notebooks/qg4Dvar_tutorial.ipynb:82]: The `\bf` markup also isn't working.

File: notebooks/qg4Dvar_tutorial.ipynb (hunk 1 above)
Comment (svahl991) [reply notebooks/qg4Dvar_tutorial.ipynb:37]: Note I also added the word `to` to this line

**Comment by cmgas** [comment]
Closing this for now @svahl991 , we'll update all the tutorials when needed :)

**Pull request closed without merging**